engine/robot.py

```diff
@@ -51,7 +46,7 @@ class Robot:
     heading is in range [0..359]
     """
 
-    def __init__(self, x_pos, y_pos, heading, epsilon, max_v, radius, is_sim=True, position_kp=1, position_ki=0,
+    def __init__(self, x_pos, y_pos, heading, epsilon, max_v, radius, is_sim=True, is_store=False, position_kp=1, position_ki=0,
```

Comment:
that's a good point! hm i didn't consider this before. I'm going to say this is ok since a lot of the storing rn is for simulation purposes + when we run the robot, we will probably want to optimize storage